lib/chunks.js

```diff
@@ -604,16 +604,25 @@ module.exports.deleteChunks = async function (courseId, chunksToDelete) {
  * the "course" directory is the reconstructed directory hierarchy
  * that mimics the source repo.
  *
+ * IMPORTANT: we previously differentiated between `base` and `course` - that
+ * is, `course` was a subdirectory of `base`. However, we've since changed
+ * that so that the base directory *is* the course directory, and all
+ * chunk-related directories are subdirectories of the course directory. This
+ * is crucial for the way that we containerize course code execution, as we
+ * need any symlinks to refer to point to something within the course directory.
+ * Otherwise, when we mount the course directory into a container, the symlinks
+ * won't be resolvable.
```

Comment:
Calling this out for review - without this, containerization + chunks won't work at all.

I don't *think* we need to do anything special here to ensure that the upgrade path works correctly. Our code should just see that these directories don't exist, create and use them, and leave the old ones alone.

Reply:
This is so that course code doesn't have access to files outside of the course?